update doc (#22)
microsoft/mysqlnd_azure: README.md

# PHP mysqlnd redirection extension mysqlnd_azure
The source code here is a PHP extension implemented using mysqlnd plugin API (https://www.php.net/manual/en/mysqlnd.plugin.php), which provides redirection feature support.  The extension is also available on PECL website at  https://pecl.php.net/package/mysqlnd_azure.

> **Important notice:**

Reviewer: Can we add a section of 'Enable redirection in server' in same level of 'Step to install using PECL' because enabling in portal is also a necessary step of enabling the feature. We need heavily project it.

Author: I chose to keep the position, but change the title to **Important notice for prerequisites to use redirection:**. it appears at the beginning of the doc, and is with bold format, so I think it may better to highlight it

Author: I've changed the level to same as Setup

> - There is a limitation that for Azure MySQL, redirection is only possible when the connection is configured with SSL, and it will only support TLS 1.2 with FIPS approved cipher for redirection.

Reviewer 2: ```suggestion
- There is a limitation for Azure DB for MySQL where redirection is only possible when the connection is configured with SSL and only works with TLS 1.2 with a FIPS approved cipher for redirection.
```

> - On server side, redirection is also need to be enabled, the setting is OFF by default. You may configure the value using server parameter **redirect_enabled**.

Reviewer: Suggest don't mention default value, as it may change from server to server. Also server created before K53 are using default on.
We need suggest Cx run show global variables  XXX and check the value on portal as well. If off, turn it on portal

Author: I've removed the part for default value, and add the way of show global variables

> - On server side, redirection is also need to be enabled, the setting is OFF by default. You may configure the value using server parameter **redirect_enabled**.

Reviewer 2: ```suggestion
- On the server side, redirection also needs to be enabled. You can enable redirection by updating the **redirect_enabled** server parameter using the [Azure portal](https://docs.microsoft.com/azure/mysql/howto-server-parameters) or [Azure CLI](https://docs.microsoft.com/azure/mysql/howto-configure-server-parameters-using-cli). This setting is OFF by default.
```
